discord/utils.py

```diff
@@ -70,6 +70,7 @@
 import types
 import warnings
 import logging
+from collections import namedtuple
```

Comment:
Ideally we should be using `typing.NamedTuple` wherever possible.

Reply:
Will do.